databricks/koalas/missing/__init__.py

```diff
@@ -18,28 +18,30 @@
 
 
 def _unsupported_function(class_name, method_name, deprecated=False, reason=""):
-
     def unsupported_function(*args, **kwargs):
-        raise PandasNotImplementedError(class_name=class_name, method_name=method_name,
-                                        reason=reason)
```

Comment:
Hmm .. this is legitimate per PEP 8 if I didn't miss anything.

Reply:
yeah, i think almost all lines in the current codebase are legitimate. This is just how Black reformat codes to standardize code style i think